modules/op-resolver-git.adoc

```diff
@@ -0,0 +1,134 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-git-specify_{context}"]
+= Specifying a remote pipeline or task using the Git resolver
+
+When creating a pipeline run, you can specify a remote pipeline from a Git repository. When creating a pipeline or a task run, you can specify a remote task from a Git repository.
+
+.Prerequisites
+
+* If you want to use teh authenticated SCM API, you configured the authenticated Git connection for the Git resolver.
```

Comment:
```suggestion
* If you want to use the authenticated SCM API, you configured the authenticated Git connection for the Git resolver.
```

(Also in case you don't know, don't use the "Commit suggestion" button to accept this change - make the change manually)